database/seeders/DefaultSkills.php

```diff
@@ -29,7 +86,42 @@ public function run(): void
             ['skill_name' => 'Electronics safety', 'category' => 2],
             ['skill_name' => 'Replacing screens', 'category' => 2],
         ];
+    }
+
+    /**
+     * Seed skills to database based on configuration
+     */
+    private function seedSkills(array $data): void
+    {
+        $truncate = env('SEEDING_TRUNCATE_SKILLS', true);
+
+        if ($truncate) {
+            Log::info('Truncating and re-inserting all skills');
+            DB::table('skills')->truncate();
+            DB::table('skills')->insert($data);
+            Log::info('Inserted ' . count($data) . ' skills');
+            return;
+        }
+
+        Log::info('Merging new skills only');
+        $added = 0;
+        $modified = 0;
+
+        foreach ($data as $entry) {
+            $skill = DB::table('skills')->where('skill_name', $entry['skill_name'])->first();
+
+            if (is_null($skill)) {
+                DB::table('skills')->insert($entry);
+                $added++;
+                continue;
+            }
+
+            if (empty($skill->description) && !empty($entry['description'])) {
```

Comment:
```suggestion
            if ($skill->description != $entry['description'] ?? '') {
```

Reply:
This has the opposite affect of what I was going for.

This will make the JSON file entry always take precedence. Meaning whatever is set in the JSON file will overwrite the DB entry, including dropping the description.

The original idea was to only modify the DB entry if its empty and we actually have a description we want to update it with. That way if we manually update the skill, it would not be overwritten.

Reply:
Okay, then you can simplify it with: `!$skill->description && $entry['description'] ?? null`

Reply:
Addressed in [103387a](https://github.com/iFixit/restarters/pull/17/commits/103387aa27045586d59f984aa491d5d7d0c44650)